css/style.css

```diff
@@ -0,0 +1,1083 @@
+}
+
+@media (min-width: 768px) {
+  .form-text-wrap {
+    padding-top: 60px;
+    width: 1440px;
+  }
+}
+
+.form-title {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: center;
+  color: #fff;
+}
+
+@media (min-width: 768px) {
+  .form-title {
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+  }
+}
+
+.form-text {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #ebebff;
+}
+
+@media (min-width: 768px) {
+  .form-text {
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    padding-left: 436px;
+    padding-right: 436px;
+  }
+}
+
+form {
+  margin: 0 auto;
+  width: 375px;
+  padding-top: 50px;
+  display: flex;
+  justify-content: center;
+}
+
+@media (min-width: 768px) {
+  form {
+    margin: 0 auto;
+    width: 1440px;
+    padding-top: 0%;
+    display: flex;
+    justify-content: center;
+  }
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+form li + li {
+  margin-top: 1em;
+}
+
+input,
+textarea {
+  font: 1em sans-serif;
+  width: 300px;
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  height: 40px;
+  text-indent: 10px;
+}
+
+@media (min-width: 768px) {
+  input,
+  textarea {
+    font: 1em sans-serif;
+    width: 447px;
+    box-sizing: border-box;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 1px;
+    height: 40px;
+    text-indent: 10px;
+    padding-top: 0%;
+  }
+}
+
+textarea {
+  left: 20px;
+  right: 20px;
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  height: 150px;
+  text-indent: 10px;
+  padding-top: 5px;
+}
+
+@media (min-width: 768px) {
+  textarea {
+    left: 20px;
+    right: 20px;
+    background: #fff;
+    border: 1px solid #cfd8dc;
+    border-radius: 1px;
+    height: 150px;
+    text-indent: 10px;
+  }
+}
+
+input:focus,
+textarea:focus {
+  border-color: #000;
+}
+
+form button {
+  width: 152px;
+  height: 54px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  background: #fff;
+  border-radius: 8px;
+  padding-left: 18px;
+  border: none;
+}
+
+@media (min-width: 768px) {
+  form button {
+    margin: 0%;
+    justify-content: center;
+    padding-left: 8px;
+  }
+}
+
+@media (min-width: 768px) {
+  .button {
+    margin: 0%;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: center;
+    margin-top: 15px;
+  }
+}
+
+button:hover {
+  background: #e65f1b;
+  color: #fff;
+  cursor: pointer;
+  transition: 1s;
+}
```

Comment:
- [ ] It is noticed that `@media (min-width: 768px)`  and `@media (max-width: 768px)`property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by designing the mobile version first and by setting the media query with  `@media (min-width: 768px)` you can design the desktop version, So Kindly Fix this issues.